charmhelpers/core/strutils.py

```diff
@@ -32,9 +34,12 @@ def bool_from_string(value):
 
     value = value.strip().lower()
 
-    if value in ['y', 'yes', 'true', 't', 'on']:
+    if value in truthy_strings:
         return True
-    elif value in ['n', 'no', 'false', 'f', 'off']:
+    elif value in falsey_strings:
+        return False
+    
```

Comment:
charmhelpers/core/strutils.py:41:1: W293 blank line contains whitespace

Reply:
:thinking: I actually added this blank line intentionally because I wanted readers to be able to keep the reporting behaviour to be mentally separate from the prior tests. I'm happy to fold them in together.

Reply:
I don't think the blank line was the issue, it was that it contained whitespace.  I like the new version better, though.